### DISCUSSION TITLE
Views being supported?

### DISCUSSION THREAD

**Discussion by thadguidry** [9tigerio/db2rest#204] [Ideas]
Also, good to support:
- We need to have a view that has 2 owners, and where we do a delete operation by 1 owner, and then a get operation by the other owner to verify the row was deleted.
- And supporting materialized views additionally. https://www.postgresql.org/docs/16/view-pg-matviews.html  and https://www.postgresql.org/docs/16/rules-materializedviews.html
- because of:
> While access to the data stored in a materialized view is often much faster than accessing the underlying tables directly or through a view, the data is not always current; yet sometimes current data is not needed.
(edited)

Views can also sometimes allow INSERT, DELETE, UPDATE
> A view can also have other kinds of DO INSTEAD rules, allowing INSERT, UPDATE, or DELETE commands to be performed on the view despite its lack of underlying storage. This is discussed further below, in [Section 41.2.4](https://www.postgresql.org/docs/16/rules-views.html#RULES-VIEWS-UPDATE).

https://www.postgresql.org/docs/16/rules-views.html (edited)

**Comment by kdhrubo** [comment]
Yes. It is supported. 
Needs testing. Hence undocumented feature at this time.

**Comment by kdhrubo** [comment]
Need to expand on the docs now. Moving to issues.
